src/NLog/LayoutRenderers/DateLayoutRenderer.cs

```diff
@@ -75,8 +75,11 @@ public DateLayoutRenderer()
         [DefaultValue(false)]
         public bool UniversalTime { get; set; }
 
-        private static char[] lowTimeResolutionChars = new char[] { 'Y', 'y', 'M', 'D', 'd', 'H', 'h' };
+        private const string lowTimeResolutionChars = "YyMDdHh";
```

Comment:
Well this is one time operation, and when below 100 elements then an array-scan is faster than calculating hash-code and doing index-lookup.